messages.proto

```diff
@@ -285,6 +285,40 @@ message StepPositionsResponse {
     string error = 2;
 }
 
+/// Request for getting Implementation file list
+message ImplementationFileListRequest {
+    /// Get the implementation directory list for folder path
+    string folderPath = 1;
+}
+
+/// Response for getting Implementation file list
+message ImplementationFileListResponse {
+    /// Get the implementation directory list for folder path
+    repeated string implementationFilePaths = 1;
+}
+
+/// Request for injecting code snippet into implementation file
+message StubImplementationCodeRequest {
+    /// File Path where code needs to be placed
+    string filePath = 1;
+
+    /// Code which needs to be placed in the file
+    string code = 2;
+}
+
+
+/// Response for injecting code snippet into implementation file
+message StubImplementationCodeResponse {
+    /// File Path where code needs to be placed
+    string filePath = 1;
+
+    /// Content which needs to be replaced in the file
+    string content = 2;
```

Comment:
File Path and file content are already there in FileChanges message type. We should be reusing that.